libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -26,6 +26,7 @@
 #include <uavcan/equipment/actuator/Command.hpp>
 #include <uavcan/equipment/actuator/Status.hpp>
 #include <uavcan/equipment/esc/RawCommand.hpp>
+#include <uavcan/equipment/esc/Status.hpp>
```

Comment:
Why is this needed?

Reply:
I put the reason in the commit text. This is to have in the future an access to ESC current consumption, voltage, rpm, node id, analog input ...

Reply:
Then please add it when it will be used.

Reply:
ok, I'll cancel that change

Reply:
revert pushed

Reply:
revert pushed